resources/assets/lib/news-index/post-item.tsx

```diff
@@ -21,39 +21,46 @@ import { route } from 'laroute';
 import * as moment from 'moment';
 import * as React from 'react';
 
-export default function PostItem({post}: {post: PostJson}) {
-  let cover;
+interface Props {
+  modifiers?: string[];
+  post: PostJson;
+}
 
-  if (post.first_image != null) {
-    cover = <img className='news-card__cover' src={post.first_image} />;
-  }
+export default class PostItem extends React.Component<Props, {}> {
```

Comment:
not sure why this is now a class, it doesn't have a constructor, nor use any lifecycle methods other than `render`